llvm/include/llvm/IR/RuntimeLibcalls.def

```diff
@@ -284,10 +284,7 @@ HANDLE_LIBCALL(FLOOR_F64, "floor")
 HANDLE_LIBCALL(FLOOR_F80, "floorl")
 HANDLE_LIBCALL(FLOOR_F128, "floorl")
 HANDLE_LIBCALL(FLOOR_PPCF128, "floorl")
-HANDLE_LIBCALL(COPYSIGN_F32, "copysignf")
-HANDLE_LIBCALL(COPYSIGN_F64, "copysign")
 HANDLE_LIBCALL(COPYSIGN_F80, "copysignl")
```

Comment:
Weird to leave the l case. What was the PPC issue? We should be able to expand x86_fp80, ppcf128, and fp128 just fine?